pyproject.toml

```diff
@@ -1,6 +1,54 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
-]
+requires = ["setuptools", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+
+[project]
+name = "twoaxistracking"
+authors = [
+    {name = "Adam R. Jensen", email = "adam-r-j@hotmail.com"},
+    {name = "Kevin S. Anderson"},
+]
```

Comment:
This results in the wheel's `METADATA` file including this, which doesn't seem quite right to me:

```
Author: Kevin S. Anderson
Author-email: "Adam R. Jensen" <adam-r-j@hotmail.com>
```

The v0.2.5 wheel on PyPI has this, which seems better to me:

```
Author: 'Adam R. Jensen, Kevin Anderson'
Author-email: adam-r-j@hotmail.com
```

Reply:
This seems like a strange behavior, see explanation below from this [link](https://packaging.python.org/en/latest/specifications/pyproject-toml/). I've made my email in a separate email entry now.

>If only name is provided, the value goes in [Author](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-author) or [Maintainer](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-maintainer) as appropriate.

>If only email is provided, the value goes in [Author-email](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-author-email) or [Maintainer-email](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-maintainer-email) as appropriate.

>If both email and name are provided, the value goes in [Author-email](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-author-email) or [Maintainer-email](https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata-maintainer-email) as appropriate, with the format {name} <{email}>.

>Multiple values should be separated by commas.

Reply:
In the `twoaxistracking.egg-info/PKG-INFO` file this now lists as:

> Author: Adam R. Jensen, Kevin S. Anderson
> Author-email: adam-r-j@hotmail.com